app/presenters/explorer_presenter.rb

```diff
@@ -253,7 +253,7 @@ def for_render_default
     data[:ajaxUrl] = ajax_action_url(@options[:ajax_action]) if @options[:ajax_action]
     data[:clearGtlListGrid] = !!@options[:clear_gtl_list_grid]
     data[:setVisibility] = @options[:set_visible_elements]
-    data[:rightCellText] = @options[:right_cell_text] if @options[:right_cell_text]
+    data[:rightCellText] = CGI.escapeHTML(@options[:right_cell_text]) if @options[:right_cell_text]
```

Comment:
Because it's RoR's standard way for doing what you want to do, it does other useful things
besides calling `CGI.escapeHTML()` and finally because we already use it in most places
in our code.